Create russian.sources.xml

doglover3920 opened the pull request:
Separate the russian sources

AbuBaniaz commented:
We will need to update the bitbake file so that it includes the new xml files
https://github.com/oe-alliance/oe-alliance-core/blob/4.4/meta-oe/recipes-oe-alliance/enigma2-plugins/epgimport-rytec.bb#L22

doglover3920 commented:
Both the Russian and Koala Spain sources are still included in the Rytec sources.
I just added them because they are not from us.  I get requests to include them.  
Once they are included in the build process I can delete them from the Rytec sources.
Now I added them as a separate file, so in the future they can update it themselves.
I did not know what was involved in adding them.  Pls. look into it to add them.  But maybe as a separate install file?
